guides/introduction/consuming.md

```diff
@@ -1,27 +1,123 @@
 # Consuming media data
 
-Other than just forwarding, we probably would like to be able to use the media right in the Elixir app to
-e..g feed it to a machine learning model or create a recording of a meeting.
+Other than just forwarding, we would like to be able to use the media right in the Elixir app to e.g.
+use it as a machine learning model input, or create a recording of a meeting.
 
-In this tutorial, we are going to build on top of the simple app from the previous tutorial by, instead of just sending the packets back, depayloading and decoding
-the media, using a machine learning model to somehow augment the video, encode and payload it back into RTP packets and only then send it to the web browser.
+In this tutorial, we are going to depayload and decode received video data to use it for ML inference.
```

Comment:
We have already introduced RTP term in the previous tutorial so maybe let's not introduce another term just at the beginning?


```suggestion
In this tutorial, we are going to unpack RTP to get the actual audio and video frames ready to be decoded and used for ML inference.
```